docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -11,6 +11,15 @@ This setup can be useful when you already have an existing Teleport cluster and
 like to start running it in Kubernetes, or when migrating your setup from a legacy
 version of the Helm charts.
 
+<Admonition type="tip" title="Have an existing Teleport cluster?">
```

Comment:
The combination of two admonitions and a notice here is a bit hard for me to read. Is there something else we can do?

Reply:
I'm adding a commit to space out the callout blocks as makes sense within the surrounding copy.

Reply:
For me, the issue is that there is kind of a distracting mix of styles competing for my attention:

<img width="943" alt="image" src="https://user-images.githubusercontent.com/25250668/228319033-b5075806-15cf-46b1-bafe-81db5de87435.png">